rhino.compute/py/SampleTkinter/makemesh.py

```diff
@@ -0,0 +1,50 @@
+from rhino3dm import *
+import tkinter as Tkinter
+import time
+import compute_rhino3d.Mesh
+import compute_rhino3d.Util
+
+auth_token = ""
+top = Tkinter.Tk()
+def onClickOk():
```

Comment:
Can this just be deleted?

Reply:
Yep! Everything down to and including `compute_rhino3d.Util.authToken = auth_token` can be chucked.